.pre-commit-config.yaml

```diff
@@ -1,33 +1,33 @@
 repos:
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v3.2.0
-  hooks:
-  - id: trailing-whitespace
-  - id: end-of-file-fixer
-  - id: check-yaml
-  - id: check-case-conflict
-  - id: check-json
-  - id: check-merge-conflict
-  - id: debug-statements
-  - id: detect-aws-credentials
-- repo: https://github.com/psf/black
-  rev: 22.3.0
-  hooks:
-  - id: black
-    language_version: python3
-    args: ["--config", "pyproject.toml"]
-- repo: https://github.com/pycqa/isort
-  rev: 5.9.3
-  hooks:
-  - id: isort
-    args: ["--profile", "black", "-l", "119"]
-- repo: https://github.com/pycqa/flake8
-  rev: "3.9.2"
-  hooks:
-  - id: flake8
-    args: ["--max-line-length=119", "--extend-ignore=E203"]
-- repo: https://github.com/pycqa/bandit
-  rev: "1.7.1"
-  hooks:
-  - id: bandit
-    args: ["-r", "-ll", "-ii"]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Your PRs have been doing this to all the pre-commit configs recently. Did you install a new formatter or something?

Reply:
Not intentionally-- I did see that someone else's PR to fix the same pre-commit issue also added a bunch of spaces to it. I can try to figure out how to undo it if that's preferable!